test/Dialect/AIG/longest-paths.mlir

```diff
@@ -0,0 +1,99 @@
+// RUN: circt-opt %s --split-input-file --pass-pipeline='builtin.module(aig-print-longest-path-analysis{test=true})' --verify-diagnostics
+
+// expected-remark-re @+2 {{fanOut=Object($root.x[0]), fanIn=Object($root.a[0], delay=4, history=[{{.+}}])}}
+// expected-remark-re @+1 {{fanOut=Object($root.x[0]), fanIn=Object($root.b[0], delay=4, history=[{{.+}}])}}
+hw.module private @basic(in %a : i1, in %b : i1, out x : i1) {
+  // These operations are considered to have a delay of 1.
+  %p = aig.and_inv not %a, %b : i1 // p[0] := max(a[0], b[0]) + 1 = 1
+  %q = comb.and %p, %a : i1  // q[0] := max(p[0], a[0]) + 1 = 2
+  %r = comb.or %q, %a : i1  // r[0] := max(q[0], a[0]) + 1 = 3
+  %s = comb.xor %r, %a : i1 // s[0] := max(r[0], a[0]) + 1 = 4
+  hw.output %s : i1
+}
+
+// expected-remark-re @+4 {{fanOut=Object($root.x[0]), fanIn=Object($root.a[0], delay=1, history=[{{.+}}])}}
+// expected-remark-re @+3 {{fanOut=Object($root.x[0]), fanIn=Object($root.b[0], delay=1, history=[{{.+}}])}}
+// expected-remark-re @+2 {{fanOut=Object($root.x[1]), fanIn=Object($root.a[1], delay=1, history=[{{.+}}])}}
+// expected-remark-re @+1 {{fanOut=Object($root.x[1]), fanIn=Object($root.b[1], delay=1, history=[{{.+}}])}}
```

Comment:
Could `@below` work for these, instead of counting the number of lines?

Reply:
Yeah, thanks!